readme.txt

```diff
@@ -190,6 +190,19 @@ Yes. Easy Digital Downloads also includes default support for Amazon Payments an
 
 == Changelog ==
 
+= 2.9.7, September 10, 2018 =
+* Fix: The edd_cart_amount data attributes were not updated when quantities were changed at checkout.
+* Fix: Updated the zip/postal code validation for Mongolia.
+* Fix: Updating a download meta using the EDD_Download class to `0` would not save correctly.
```

Comment:
I'd change this to read:
`Updating a download meta to '0' using the EDD_Download class would not save correctly`